lib/simmons.rb

```diff
@@ -57,37 +58,51 @@ def parse_message(message)
         Hash[message.split('; ').map { |s| s.split(': ') }]
       end
 
-      def send_to_dreamcomfort(lead)
-        customer = lead.customer
-        source_name = lead.source.name
-        source_name = if source_name.downcase.include?('widgrid')
-                        source_name.split(' - ').map(&:capitalize)[0..1].reverse.join(' - ')
-                      else
-                        source_name.split(' - ').reverse.join(' - ')
-                      end
+      def handle_integrated_stores(store_group)
+        integrated_stores = %w[dreamcomfort confortale]
+        send("send_to_#{store_group}") if integrated_stores.include?(store_group)
+      end
+
+      def send_to_dreamcomfort
+        post_to_integrated_store('simmonsdreamcomfort', parse_message_to_dreamcomfort(@lead.message))
+      end
+
+      def send_to_confortale
+        post_to_integrated_store('confortalecolchoes', @lead.message)
+      end
 
+      def post_to_integrated_store(store_group, message)
+        customer = @lead.customer
 
         HTTP.post(
-          'https://simmonsdreamcomfort.f1sales.org/public/api/v1/leads',
+          "https://#{store_group}.f1sales.org/public/api/v1/leads",
           json: {
             lead: {
-              message: parse_message_to_dreamcomfort(lead.message),
+              message: message,
               customer: {
                 name: customer.name,
                 email: customer.email,
                 phone: customer.phone
               },
               product: {
-                name: lead.product.name
+                name: @lead.product.name
               },
               source: {
-                name: source_name
+                name: parse_source(@lead.source.name)
               }
             }
           }
         )
       end
 
+      def parse_source(source_name)
```

Comment:
I think this method name is not clear.
`parse_source_to_integrated_store` would be more clear but too long.
Do you have any suggestions?